src/lib/core/components/visualizations/implementations/MosaicVisualization.tsx

```diff
@@ -801,10 +801,7 @@ function MosaicViz(props: Props<Options>) {
         controlsNode={controlsNode}
         tableGroupNode={tableGroupNode}
         // statistics tab is disabled in 2x2, so no need to prompt for required inputs
```

Comment:
Let's remove this comment too, since the Stats tab won't be disabled anymore.

Reply:
yep will remove it 👍